src/test/scala/zio/codec/examples/CodecJsonTest.scala

```diff
@@ -0,0 +1,32 @@
+package zio.codec.examples
+
+import zio.Chunk
+import zio.codec.{ CharCodecModule, DecodeError, Equiv }
+
+object CodecJsonTest {
+
+  object JsonCodec extends CharCodecModule
+
+  import Equiv._
+  import JsonCodec._
+
+  def char(c: Char): Codec[Char] =
+    consume.filter(Set(c))
+
+  val colon: Codec[Char] = char(':')
+  val quote: Codec[Char] = char('"')
+
+  val spacing: Codec[Unit] =
+    consume.filter(Set(' ', '\n', '\r')).rep.ignore(Chunk.empty)
```

Comment:
We should add helpers like:

```scala
oneOf(' ', '\n', '\r')
```

etc. which are defined in terms of the base codecs.

Reply:
https://github.com/zio/zio-codec/issues/13